examples/rest-js/dub.json

```diff
@@ -0,0 +1,8 @@
+{
+	"name": "rest-js-example",
+	"description": "Simple dynamic web UI that accesses a REST interface",
+	"dependencies": {
+		"vibe-d": { "path": "../../" }
+	},
+	"versions": ["VibeDefaultMain"]
+}
```

Comment:
Nitpick: Missing newline.

As per the example that follow, there's no comment so it's a bit hard on newcomers to see how useful it is.